src/sentry/utils/performance_issues/performance_detection.py

```diff
@@ -229,18 +229,17 @@ def get_detection_settings(project_id: int | None = None) -> dict[DetectorType,
     settings = get_merged_settings(project_id)
 
     return {
-        DetectorType.SLOW_DB_QUERY: [
-            {
-                "duration_threshold": settings["slow_db_query_duration_threshold"],  # ms
-                "allowed_span_ops": ["db"],
-                "detection_enabled": settings["slow_db_queries_detection_enabled"],
-            },
-        ],
+        DetectorType.SLOW_DB_QUERY: {
```

Comment:
Flattening these all to be consistent, the list wasn't necessary since they all only had one element anyway